tests/gptq/test_quantization.py

```diff
@@ -178,7 +187,7 @@ def test_exllama_serialization(self):
                 )
             empty_model.tie_weights()
             quantized_model_from_saved = load_quantized_model(
-                empty_model, save_folder=tmpdirname, device_map={"": 0}, disable_exllama=False
+                empty_model, save_folder=tmpdirname, device_map={"": 0}, disable_exllama=False, disable_exllamav2=True
```

Comment:
I would add a `@parameterized.expand` here to test both exllama and exllama v2. Edit: but the approach with two classes works (though it duplicates code).

Reply:
I prefer to keep the duplicated code. The two classes are not exactly the same for exllama and exllamav2. For exllama, we need to differentiate between act-order and no act-order whereas it is not the case for exllamav2.